.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+  "*.{ts,tsx}": ["prettier --parser typescript --single-quote --write", "git add"]
```

Comment:
Do we need `--parser typescript` if it infers it from file path. 

https://github.com/prettier/prettier#parser

I don't have anything against being explicit